utils/ccdatofhir/ccda_to_fhir.bal

```diff
@@ -0,0 +1,186 @@
+                        xml<xml:Element> patientRoleElement;
+                        if isNamespaceAvailable {
+                            patientRoleElement = xmlContent.children().<v3:patientRole>;
+                        } else {
+                            patientRoleElement = xmlContent.children().<patientRole>;
+                        }
+                        r4:Patient|error? mapCCDAPatientToFHIRResult = mapCcdaPatientToFhir(patientRoleElement, isNamespaceAvailable);
+                        if mapCCDAPatientToFHIRResult is error {
+                            return logAndThrowError(mapCCDAPatientToFHIRResult);
+                        }
+                        entries.push({'resource: mapCCDAPatientToFHIRResult});
+                    }
+                }
+                xml<xml:Element|xml:Comment|xml:ProcessingInstruction|xml:Text> authorElement = xmlPayload/<v3:author>;
+                if authorElement is xml:Element {
+                    r4:Practitioner|error? mapCCDAToFHIRResult = mapCcdaPractitionerToFhir(authorElement, isNamespaceAvailable);
+                    if mapCCDAToFHIRResult is error {
+                        return logAndThrowError(mapCCDAToFHIRResult);
+                    }
+                    if mapCCDAToFHIRResult.toJson() != () {
+                        entries.push({'resource: mapCCDAToFHIRResult});
+                    }
+                }
+
+                xml<xml:Element> componentElements = xmlPayload/<v3:component>/<v3:structuredBody>/<v3:component>;
+                isNamespaceAvailable = true;
+                if componentElements.length() == 0 {
+                    isNamespaceAvailable = false;
+                    componentElements = xmlPayload/<component>/<structuredBody>/<component>;
+                }
+                if componentElements.length() > 0 {
+                    foreach xml:Element componentElement in componentElements {
+                        xml<xml:Element> sectionElement;
+                        if isNamespaceAvailable {
+                            sectionElement = componentElement/<v3:section>;
+                        } else {
+                            sectionElement = componentElement/<section>;
+                        }
+
+                        if sectionElement is xml:Element {
+                            xml<xml:Element> codeElement;
+                            if isNamespaceAvailable {
+                                codeElement = sectionElement/<v3:code>;
+                            } else {
+                                codeElement = sectionElement/<code>;
+                            }
+
+                            if codeElement is xml:Element {
+                                string codeVal = check codeElement.code;
+                                xml<xml:Element>|xml<xml:Comment>|xml<xml:ProcessingInstruction>|xml<xml:Text> entryElements = (isNamespaceAvailable) ? sectionElement/<v3:entry> : sectionElement/<entry>;
```

Comment:
Why are you using parenthesis around `isNamespaceAvailable`?

Reply:
Removed in https://github.com/ballerina-platform/module-ballerinax-health.fhir.r4/commit/817dadb5178bde7d809333a26f465c6688d3879a